storage/remote/generic/generic.proto

```diff
@@ -15,9 +15,8 @@ syntax = "proto3";
 
 package generic;
 
-
 message Sample {
-  double value = 1;
+  double value       = 1;
   int64 timestamp_ms = 2;
```

Comment:
It's the same naming as in https://github.com/prometheus/client_model/blob/master/metrics.proto#L73, probably best to stay consistent?